zplugin-ros1/src/config.rs

```diff
@@ -0,0 +1,160 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+use regex::Regex;
+use serde::{de, Deserialize, Deserializer};
+use std::env;
+use std::time::Duration;
+use zenoh::prelude::*;
+
+pub const DEFAULT_DOMAIN: u32 = 0;
+pub const DEFAULT_GROUP_LEASE_SEC: f64 = 3.0;
```

Comment:
Why not defining the const directly with Duration type?
```rust
pub const DEFAULT_GROUP_LEASE: Duration = Duration::from_secs(3.0);
```

Reply:
Oops, this whole file is unnecessary =)